gnd/src/main/java/com/google/android/gnd/system/StorageManager.java

```diff
@@ -88,23 +90,26 @@ private Completable sendPhotoPickerIntent() {
   public Observable<Bitmap> photoPickerResult() {
     return activityStreams
         .getNextActivityResult(PICK_PHOTO_REQUEST_CODE)
-        .flatMap(this::onPickPhotoResult)
-        .map(uri -> Media.getBitmap(context.getContentResolver(), uri));
+        .flatMapMaybe(this::onPickPhotoResult)
```

Comment:
Please annotation this method with `@Hot`.

Reply:
Done